backend/pyproject.toml

```diff
@@ -39,4 +39,8 @@ strict = ["*"]
 dev-dependencies = [
     "pyright>=1.1.387",
     "ruff>=0.7.2",
+    # Add the type stubs here
+    "types-flask",
+    # "types-flask-login",- No solution found when resolving dependencies for split (python_full_version == '3.12.*')
+    "types-sqlalchemy"
```

Comment:
we should not have either of these, as they are built for older versions of flask/sqlalchemy and mess up the typechecker further